### DISCUSSION TITLE
Permissions error

### DISCUSSION THREAD

**Discussion by OpenSource03** [pterodactyl/panel#3949] [Support]
Hello there!

I'm experiencing an error:

`The stream or file "/home/vagrant/app/storage/logs/laravel-xxxx-xx-xx.log" could not be opened in append mode: Failed to open stream: Permission denied`

Of course, I'm aware that it's a permission error and the following commands should fix it:

```
chmod -R 755 storage/* bootstrap/cache/
chown -R www-data:www-data /var/www/pterodactyl/*
```

Unfortunately, even after applying them, it's only a temporary fix as the issue comes back a few hours/days after when I need to do the commands again. Is there a workaround for that?

**Comment by trenutoo** [comment]
Someone is editing the files if their permissions are changing after a few hours. For example, you're modifying the panel files and changing the owner or permissions.

**Comment by TheFruxz** [comment]
I experience the exact same issue here too
No idea how to fix that, but leads to way more issues and kinda rendering pterodactyl useless
